Gopkg.toml

```diff
@@ -97,3 +97,7 @@ required = [
 [[constraint]]
   branch = "master"
   name = "github.com/openshift/client-go"
+
+[[constraint]]
+  name = "k8s.io/kubernetes"
+  version = "1.15.1"
```

Comment:
This will be a nack from @shawn-hurley. 

We cannot move to 1.15 before rest of OpenShift. We actually should be pinning to  kube version to 1.14. Do you need 1.15.1 to use `labels.CloneAndRemoveLabel()`? I see it in the [vendor folders](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/vendor/k8s.io/kubernetes/pkg/util/labels/labels.go#L41) in OLM and they are pinned to 1.14. 

@ecordell please correct if I am wrong?

Reply:
Ah that's my bad I just did `dep ensure --add ...` - what is the correct way of adding this dependency such that it doesn't bump to 1.15?

Reply:
```
dep ensure --add --help
Usage: dep ensure [-update | -add] [-no-vendor | -vendor-only] [-dry-run] [-v] [<spec>...]

Project spec:

  <import path>[:alt source URL][@<constraint>]


Ensure gets a project into a complete, reproducible, and likely compilable state:

  * All imports are fulfilled
  * All rules in Gopkg.toml are respected
  * Gopkg.lock records immutable versions for all dependencies
  * vendor/ is populated according to Gopkg.lock

Ensure has fast techniques to determine that some of these steps may be
unnecessary. If that determination is made, ensure may skip some steps. Flags
may be passed to bypass these checks; -vendor-only will allow an out-of-date
Gopkg.lock to populate vendor/, and -no-vendor will update Gopkg.lock (if
needed), but never touch vendor/.

The effect of passing project spec arguments varies slightly depending on the
combination of flags that are passed.


Examples:

  dep ensure                                 Populate vendor from existing Gopkg.toml and Gopkg.lock
  dep ensure -add github.com/pkg/foo         Introduce a named dependency at its newest version
  dep ensure -add github.com/pkg/foo@^1.0.1  Introduce a named dependency with a particular constraint
```
:point_up:

Reply:
Please make it a separate commit if possible.